package.json

```diff
@@ -41,6 +42,7 @@
     "husky": "^4.2.5",
     "jest": "^25.5.4",
     "lint-staged": "^10.2.2",
+    "memory-fs": "^0.5.0",
```

Comment:
`memory-fs` is already deprecated. You can use `memfs` instead, but it doesn't a proper error message, so I use `memory-fs` at the time. Here is the log that I used `memfs`.
https://circleci.com/gh/kufu/smarthr-ui/5966

I'll investigate this later.